omero/autogen_docs

```diff
@@ -51,6 +51,7 @@ for f in \
     omero-* \
     settings.env \
     setup_* \
+    docker_virtualenv*\
```

Comment:
We have one link pointing to one the file (Apache24 support)
I can remove it and point to the omero-install instead so it is not included in the doc.